schema.graphql

```diff
@@ -242,18 +255,28 @@ type Settlement @entity {
   eventId: String!
   addresses: [String!]! @index(unique: false)
   result: String!
-  legs: [Leg!]!
+  legs: [Leg!]! @derivedFrom(field: "settlement")
+}
+
+type Venue @entity {
+  id: ID! # venueId
+  identity: Identity!
```

Comment:
```suggestion
  owner: Identity!
```

thoughts?

Reply:
I'm OK with both. I'll change to owner

Reply:
I think it reads a bit easier for the person making a query.